### ISSUE TITLE
[Suggestion] Animeated Dynamos

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [CoFH/Feedback#681]
https://github.com/elytra/DynamicDynamos

Should be an optional feature via config file. I like animated stuff.

**Comment by KingLemming** [comment]
It's been on the the todo list for a long time, but the fact that another mod is doing it makes me far less likely to bother.

And it'd be rude to effectively destroy that mod. So, not super likely.

**Comment by Unrepentant-Atheist** [comment]
I dunno about destroy, incorporate it and credit the author, but having yet another mod to load just for aesthetic purposes is silly in my opinion. I'Ve far too many mods loaded as is.

**Comment by AnZaNaMa** [comment]
This may not be as simple as it seems, though, because by the  license of Thermal Expansion, once something is incorporated into the mod it becomes the intellectual property of CoFH and the author of that mod may not be okay with that.

**Comment by KingLemming** [comment]
Exactly. I'm actually looking at adjusting the license a bit to have a formalized CLA process, but the general fact of us owning the code will remain.

**Comment by KingLemming** [comment]
Dynamos now have animated textures. I'm on the fence with a piston. We'll see.

**Comment by Drullkus** [comment]
I absolutely love a piston.

What I really thought was the coolest ever was RP2’s engine, that had a gear too! Not feasible on all Dynamo types, of course.

**Comment by covers1624** [comment]
Its something to consider, but imo, forge's animation pipeline is horrible, not to mention the lighting system. CCL has a wayy faster pipeline. I have ideas's for a modular CCL, where modules could be jar in jar packed with the new dependency system, if that happens, its possible i'd look in to my own more optimised animation system built on top of CCL's render pipeline. Then the possibility of having dynamo's animated would be an option.
